src/Tasks/Resources/Strings.resx

```diff
@@ -2745,6 +2745,27 @@
   <data name="ZipDirectory.Comment">
     <value>Zipping directory "{0}" to "{1}".</value>
   </data>
+
+  <!--
+        MSB3951 - MSB3960  Tasks: GetFileHash and VerifyFileHash
+        If this bucket overflows, pls. contact 'vsppbdev'.
+  -->
+  <data name="VerifyFileHash.InvalidInputParameters" xml:space="preserve">
+    <value>MSB3951: Invalid input parameters. Either Hash or HashBase64 must be specified, but not both.</value>
+    <comment>{StrBegin="MSB3951: "}</comment>
+  </data>
+  <data name="VerifyFileHash.HashMismatch" xml:space="preserve">
+    <value>MSB3952: File hash mismatch. Expected {0} to have a {1} file hash of {2}, but it was {3}.</value>
+    <comment>{StrBegin="MSB3952: "}</comment>
+  </data>
+  <data name="FileHash.UnrecognizedHashAlgorithm" xml:space="preserve">
+    <value>MSB3953: Unrecognized hash algorithm name '{0}'.</value>
+    <comment>{StrBegin="MSB3953: "}</comment>
+  </data>
+  <data name="FileHash.FileNotFound" xml:space="preserve">
+    <value>MSB3954: failed to compute file hash '{0}'. File not found.</value>
```

Comment:
```diff
-MSB3954: failed to compute file hash '{0}'. File not found.
+MSB3954: Failed to compute hash for file '{0}' because it does not exist or is inaccessible.
```